### ISSUE TITLE
nc-ng suddenly killed

### ISSUE THREAD

**Issue by patricevenant** [eteran/nedit-ng#77]
Hi Evan,
The nc-ng stops working suddenly. This happens time to time. I suspect the communication between Qt and nc-ng fails. I haven't any message. Do you have any idea where I can produce a log to help to diagnostic ?

By curiosity, I run nedit-ng in same condition and I got the following kind of message without losing my nedit-ng process : 
Warning: QXcbConnection: XCB error: 3 (BadWindow), sequence: 61656, resource id: 18992964, major code: 40 (TranslateCoords), minor code: 0 ((null):0, (null))
Any clue about nc-ng ?

**Comment by eteran** [comment]
So the XCB warnings are produced by Qt itself. I'm not if there is anything that can or even should be done about it. But in general, they should be rare.

As for nc-ng, do you mean that it does literally nothing?

The link between nc-ng and nedit-ng is via a UNIX local socket (usually a pipe somewhere in `/var/run/...` , so the only thing that I would expect can go wrong is if the QLocalSocket in the "server" (nedit-ng) somehow is either dead or unable to make progress, both are cases that I've yet to see happen.

Is there anything you see correlated with this issue? Is it just a matter of time, any other apps open that seem to cause it?

**Comment by patricevenant** [comment]
All windows opened by nc-ng are disappeared suddenly. I don't see any error messages. And then, I need to start again a new nedit server.

In fact, you gave me a good tack to explore. Probably another apps disrupts the socket pipe. Let's try to investigate that...

**Comment by eteran** [comment]
Oh, if the actual windows that we shown close .. that sounds like a plain old crash unrelated to nc-ng (nc is really just a remote control for nedit-ng)

**Comment by patricevenant** [comment]
The crash of nc-ng arrives when I play with the open file dialog. In my case, I firstly, launch nc-ng without argument and then I look for the file ~/.config/nedit-ng/config.ini through the open file dialog. When I've  pushed on open button, nc-ng is closing immediately (even if file is in modified state).  I don't have any message on my console. :(

I've tried the same flow with nedit-ng and I got : 
Debug: using qt5ct plugin ((null):0, (null))
Debug: D-Bus global menu: no ((null):0, (null))
Warning: QXcbConnection: XCB error: 3 (BadWindow), sequence: 1199, resource id: 20996572, major code: 40 (TranslateCoords), minor code: 0 ((null):0, (null))


Have you any clue to produce a better report of the issue ?
